blinkit-app/src/homePage/molecules/logoLocation/logoLocation.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../../mixin';
+
+
+.blinkit-logo:hover{
+    cursor: pointer;
+}
+.logo-location{
+    width: 22rem;
+    @include flexProp(space-between,center,wrap);
+}
+.logo{
+    padding-right: 0.5rem;
+    border-right: 0.1rem solid rgb(238, 231, 231);
+}
+.location{
+    div{
```

Comment:
Never style DOM elements directly, unless unavoidable. Pass in classnames to the respective tags as per your needs